styleguide/components/31-molecules/building-blocks/breadcrumbs/_breadcrumbs.scss

```diff
@@ -2,46 +2,42 @@
   ul,
   ol {
     display: none;
-    margin: .5rem auto .5rem -.35rem;
+    line-height: normal;
```

Comment:
Is browser dependent.

Reply:
Turns out this wasn't necessary. I removed it